controllers/google-api-controller.js

```diff
@@ -393,9 +393,63 @@ exports.geocode = (appReq, appRes) => {
     res.on('end', () => {
       const body = Buffer.concat(chunks);
       const queryResult = JSON.parse(body.toString());
-      appRes.status(200).send(queryResult); 
+      appRes.status(200).send(queryResult);
     });
   }).on('error', (err) => {
     appReq.send(err);
   });
 };
+
+/**
+* TextSearch API
+* A Text Search request is an HTTP URL of the following form:
+* https://maps.googleapis.com/maps/api/place/textsearch/output?parameters
+*
+* Where output may be either of the following values:
+* - json (recommended) indicates output in JavaScript Object Notation (JSON)
+* - xml indicates output as XML
+*/
+/**
+* @description Handles request for search input query to get back search results and its geocode details
+*
+* @api {POST} /search/textSearch
+* @apiSuccess 200 {JSON} With two root elements:
```

Comment:
The docs needs a little clean up - seems to have information from what you copied from.